plugin/trino-base-jdbc/pom.xml

```diff
@@ -73,6 +73,11 @@
             <artifactId>jackson-annotations</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Dependency addition? Why?

Reply:
If removed there will be error `type <xxx> not supported by default: add Module \"com.fasterxml.jackson.datatype:jackson-datatype-jsr310\" to enable handing (through...`

Reply:
Removed since we don't need `LocalDateTime`